eyed/SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
```

Comment:
can you remove the root files that are not related to the theme ?
SECURITY, CODE_OF_CONDUCT, .gitignore

Reply:
Yes sure